mllib/src/main/scala/org/apache/spark/ml/param/sharedParams.scala

```diff
@@ -34,6 +34,43 @@ private[ml] trait HasRegParam extends Params {
   def getRegParam: Double = get(regParam)
 }
 
+private[ml] trait HasElasticNetParam extends HasRegParam {
```

Comment:
I voted for `regAlphaParam: Double` for the variance name, and keep `HasElasticNetParam` as the trait name. What do you think?